Day-II-Calculator/calculator/src/components/ButtonComponents/OperatorContainer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './OperatorContainer.css';
+import OperatorButton from './OperatorButton';
+
+const buttons = [
+    {
+        text:'/',
+        style:'operator-button'
+    },
+    {
+        text:'*',
+        style:'operator-button'
+    },
+    {
+        text:'+',
+        style:'operator-button'
+    },
+    {
+        text:'-',
+        style:'operator-button'
+    },
+    {
+        text:'=',
+        style:'operator-button'
+    },
```

Comment:
I feel like this could be shortened as well. For example:
```javascript
const btns = '/*+-='.split('').map(op => {
  return {
    text: op,
    style: 'operator-button'
  };
});
```